src/Bridge/Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -83,6 +83,10 @@ public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGenerator
         }
 
         $itemsPerPage = $resourceMetadata->getCollectionOperationAttribute($operationName, 'pagination_items_per_page', $this->itemsPerPage, true);
+        if ($request->attributes->get('_graphql')) {
```

Comment:
`has()` instead of `get()`?

Reply:
`get` is safer IMO (in case it is explicitly set to `false`)

Reply:
Okay, there are a lot of places where he uses `has()` so it would also be safer to replace them!